src/tools/list/list-quotes.tool.ts

```diff
@@ -11,9 +11,10 @@ const ListQuotesTool = CreateXeroTool(
   {
     page: z.number(),
     contactId: z.string().optional(),
+    quoteNumber: z.string().optional(),
   },
-  async ({ page, contactId }: { page: number; contactId?: string }) => {
-    const response = await listXeroQuotes(page, contactId);
+  async ({ page, contactId, quoteNumber }: { page: number; contactId?: string; quoteNumber?: string }) => {
```

Comment:
The typing here can be removed as well
```suggestion
  async ({ page, contactId, quoteNumber }) => {
```